common/darwin/Classes/AudioUtils.m

```diff
@@ -74,7 +74,6 @@ + (void)setSpeakerphoneOn:(BOOL)enable {
                   :
                   AVAudioSessionCategoryOptionAllowBluetooth | AVAudioSessionCategoryOptionAllowBluetoothA2DP
                     error:nil];
-    [session setActive:YES error:nil];
```

Comment:
This causes the interruption (called in `initWithChannel`)
Calling `setActive` seems redundant as the audio session would have already been activated
Did some tests and `setSpeakerOn` seems to work fine without this line